packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -45,28 +51,65 @@ interface CreateSandboxProps {
   isModal?: boolean;
 }
 
+type EssentialsState =
+  | {
+      state: 'loading';
+    }
+  | {
+      state: 'success';
+      essentials: ITemplateInfo[];
+    }
+  | {
+      state: 'error';
+      error: string;
+    };
+
 export const CreateSandbox: React.FC<CreateSandboxProps> = ({
   collectionId,
   initialTab,
   isModal,
 }) => {
-  const { hasLogIn } = useAppState();
+  const { hasLogIn, activeTeamInfo, user } = useAppState();
   const actions = useActions();
 
+  const isRepositoriesPage = location.pathname.includes('/repositories');
+  const isDashboardPage = location.pathname.includes('/dashboard');
+  const isUser = user?.username === activeTeamInfo?.name;
```

Comment:
Can you match if the active team id is equal to the user's personal team id?

Reply:
Sadly we can't. I checked the data I have and the only other relation between the `activeTeam` and the `user` are the `user.id` and the `activeTeamInfo.creatorId`. Cant use the `creatorId`, because multiple teams can have the same creator. The only relation that is left is the `user.username` and the `activeTeamInfo.name` so I used that one.